src/main/java/com/yunus1903/chatembeds/client/screen/AnimatedImageEmbedScreen.java

```diff
@@ -41,9 +42,10 @@ public void render(MatrixStack matrixStack, int mouseX, int mouseY, float partia
             else currentFrame++;
         }
 
-        minecraft.getTextureManager().bindTexture(embed.getFrames().get(currentFrame).getResourceLocation());
-        AbstractGui.blit(matrixStack, (width - scaledImageWidth) / 2,
+        RenderSystem.setShaderTexture(0, embed.getFrames().get(currentFrame).getResourceLocation());
+        RenderSystem.enableBlend();
```

Comment:
_to be honest_ I'm not even sure how this worked before. It really seems like this should have been required the whole time unless you were taking advantage of some temporal coupling.